### PR TITLE
Give new Gmax forms accurate abilities

### PR DESCRIPTION
**Pull request by ajhyndman**
It looks like a few copy-pasted abilities snuck in.

### CODE REVIEW

File: calc/src/data/species.ts [hunk 1]
```diff
@@ -8454,7 +8454,7 @@ const SS_PATCH: {[name: string]: DeepPartial<SpeciesData>} = {
     types: ['Poison', 'Fairy'],
```
Comment (KrisXV) [inline calc/src/data/species.ts:8454]:
```suggestion
    types: ['Grass', 'Poison'],
```

File: calc/src/data/species.ts [hunk 2]
```diff
@@ -8237,7 +8237,7 @@ const SS_PATCH: {[name: string]: DeepPartial<SpeciesData>} = {
     types: ['Grass'],
     bs: {hp: 100, at: 125, df: 90, sa: 60, sd: 70, sp: 85},
     weightkg: 80,
```
Comment (KrisXV) [inline calc/src/data/species.ts:8239]:
```suggestion
    weightkg: 0,
```

Comment (ajhyndman) [reply calc/src/data/species.ts]: Looks like urshifu slipped in as well.  Good catch.

**Comment by scheibo** [comment]
Thanks, this was fixed by 397b263

**Pull request closed without merging**